template.yaml

```diff
@@ -549,7 +549,23 @@ Resources:
               "Type": "Task",
               "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Onboarding-Update-Db",
               "InputPath": "$",
-              "Next": "TriggerCloudCheckr"
+              "Next": "CloudcheckrChoiceState"
+            },
+            "CloudcheckrChoiceState": {
+              "Type" : "Choice",
+              "Choices": [
+                {
+                  "Variable": "$.cloudcheckr.isCloudcheckrEnabled",
```

Comment:
the parameter should be enableCloudCheckr as this will then enable the cloudcheckr service for the account